.circleci/config.yml

```diff
@@ -68,4 +70,8 @@ jobs:
           path: /tmp/test-results
       - store_artifacts:
           path: /tmp/test-results
-          destination: test-results
\ No newline at end of file
+          destination: test-results
+      # deploy application
+      - deploy:
```

Comment:
I think you will need you add the following before the -deploy command to get coveralls to work.  
```
- store_artifacts:
                path: coverage
```

You can tell it's working when you see your build/coverage show up here: https://coveralls.io/github/uclibs/application_portfolio